### PR TITLE
Refactor templates and row sizes in ids-data-grid

### PR DESCRIPTION
**Pull request by vivz753**
**Explain the details for making this change. What existing problem does the pull request solve?**

- Refactored templates for more readability, following AirBnB style guides
- Standardized row sizes in a local array, shortened names i.e. from 'large' to lg

**Related github/jira issue (required)**:

**Steps necessary to review your pull request (required)**:

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [ ] A note to the change log.

### CODE REVIEW

File: src/components/ids-data-grid/ids-data-grid.js [hunk 1]
```diff
@@ -90,50 +97,30 @@ class IdsDataGrid extends mix(IdsElement).with(
...
-        <ids-virtual-scroll>
-          <div class="ids-data-grid-container" part="container">
-            <div class="ids-data-grid-body" part="body" role="rowgroup" slot="contents">
-            </div>
-          </div>
-        </ids-virtual-scroll>
+      ${this.headerTemplate()}
+      ${this.virtualScroll
+      ? `<ids-virtual-scroll>
+          <div class="ids-data-grid-body" part="style-wrapper" part="body" role="rowgroup"></div>
```
Comment (tmcconechy) [inline src/components/ids-data-grid/ids-data-grid.js:120]: Two parts here? Does that work or a mistake? Prefer part="body" for the external API

**Review by tmcconechy — CHANGES_REQUESTED** [review]
Looks good. Just a few small things.

1) Two parts as noted in the inline comment (lets leave part="body")
2) Can you update ids-datagrid.d.ts as well with the new row heights
3) Can you update demos/ids-lookup/example.js with the row height change?

I was also wondering why percy changed the visual image as it seems like it should be the same? Any thoughts?

File: src/components/ids-data-grid/ids-data-grid.js (hunk 1 above)
Comment (vivz753) [reply src/components/ids-data-grid/ids-data-grid.js]: Removed the part="style-wrapper"

**Comment by vivz753** [comment]
Completed all 4 items listed above. Also changed rowHeight and row-height accordingly in a few other examples. Don't know why the styles changed in that previous iteration, but it should be gone now... waiting for test to finish.

**Review by tmcconechy — DISMISSED** [review]

File: src/components/ids-data-grid/README.md [hunk 2]
```diff
@@ -93,6 +93,7 @@ When used as an attribute the settings are kebab case, when used in the JS they
 - `alternateRowShading` {boolean} For better scan-ability you can shade alternate rows.
 - `listStyle` {boolean} Sets the style of the grid to list style for simple readonly lists.
 - `columns` {Array<object>} Sets the data array of the datagrid. This can be a JSON Array.
+- `rowHeight` {string|number} Sets the height of each row
```
Comment (andriivarhanov) [inline src/components/ids-data-grid/README.md:96]: please remove `number` here and add 'xs' | 'sm' | 'md' | 'lg' options

**Review by andriivarhanov — COMMENTED** [review]
looks good, just one minor thing

File: src/components/ids-data-grid/README.md (hunk 2 above)
Comment (vivz753) [reply src/components/ids-data-grid/README.md]: Done, thanks

**Review by andriivarhanov — APPROVED** [review]

**Review by tmcconechy — APPROVED** [review]

**Pull request merged**